settings.go

```diff
@@ -1,17 +1,28 @@
 package logger
 
 import (
+	"errors"
 	"os"
 	"strings"
 )
 
 var (
 	out                 = os.Stderr
-	verbosity           = Verbosity()
-	enabled, allEnabled = Enabled()
+	verbosity           = initVerbosity()
+	enabled, allEnabled = initEnabled()
 )
 
+// Enabled - Returns the map representing enabled logs, and whether all loggers are
+// enabled regardless of name. These are initialised from environment variables
+// at start-up.
 func Enabled() (map[string]bool, bool) {
+	return enabled, allEnabled
+}
+
+// initEnabled - Gets the LOG environment variable, splits on commas, and stores the
+// values as loggers that are permitted to print. Does not validate that loggers
+// exist; typos may break loggers silently.
+func initEnabled() (map[string]bool, bool) {
```

Comment:
how about making this just `init` and setting `enabled` and `allEnabled` from inside of `init` ?